src/main/java/org/beta/tchap/identite/authenticator/TchapAuthenticator.java

```diff
@@ -40,8 +40,9 @@ public void authenticate(AuthenticationFlowContext context) {
                     Response.status(400).build(), "request is malformed", "request is malformed" );
             return;
         }
-
-        LOG.infof("Authenticate login : %s", loginHint);
+        if(LOG.isDebugEnabled()){
```

Comment:
Note for later(this is a suggestion and not a mandatory change) : we could refactor this piece of code to have a one liner, easier to use.

`if(LOG.isDebugEnabled()){
            LOG.debugf("Authenticate login : %s", LoggingUtilsFactory.getInstance().logOrHash(loginHint));
        }`